fence/README.md

```diff
@@ -66,6 +66,11 @@ service FenceController {
   // ListClusterFence RPC call to provide a list of blocklisted/fenced clients.
   rpc ListClusterFence(ListClusterFenceRequest)
   returns (ListClusterFenceResponse){}
+
+  // GetFenceClients RPC calls to get the clients information that needs
+  // to be fenced.
```

Comment:
Maybe _.. get the client information to use in a FenceClusterNetwork or UnfenceClusterNetwork RPC._